packages/frontend/translations/en-us.yaml

```diff
@@ -352,6 +352,7 @@ general:
   reportDisplayTitleWithoutObject: "All {subject} in {school}"
   reportName: Report Name
   reportNamePlaceholder: Please enter a report name.
+  reportResultsExceedMax: Only the first 200 results shown. Download report to see all results.
```

Comment:
The 200 is hardcoded here, but it's also hardcoded in the course results. For maximum refactorability this should get passed in as `{resultsLimit}` when this translation is invoked in the `course.hbs` template.